main/src/test/java/org/mobilitydata/gtfsvalidator/report/model/FeedMetadataTest.java

```diff
@@ -120,6 +120,105 @@ public void omitsRouteNamesComponentTest3() throws IOException, InterruptedExcep
         ImmutableList.of(GtfsRouteTableDescriptor.class, GtfsAgencyTableDescriptor.class));
   }
 
+  @Test
```

Comment:
Good job on the comments explaining the tests.

Reply:
Thank you! I like writing comments - make newcomers less confused!